src/cli/tc.rs

```diff
@@ -31,9 +33,60 @@ pub async fn run() -> Result<()> {
         Err(e) => Err(e),
       }
     }
+    Command::Init { file_path } => Ok(init(&file_path).await?),
   }
 }
 
+pub async fn init(file_path: &str) -> Result<()> {
+  let tailcallrc = fs::read_to_string("assets/.tailcallrc.graphql")?;
+
+  loop {
+    println!("{}", "Do you want to add a file to the project? (yes/no/quit)".yellow());
```

Comment:
`yes` `Yes` `y` and other forms of "yes" should all work. Similarly `no` `n` or `q` should all work. Probably it's work creating a function that takes the user input and returns an enum

```rust
enum Input {
  Yes,
  No,
  Quit
}
``

Reply:
@tusharmath What do you think about using this crate for this? https://github.com/mikaelmello/inquire#Confirm

Reply:
Purfeckt!